### DISCUSSION TITLE
Max Images Stitched In One Run

### DISCUSSION THREAD

**Discussion by davidhardeniv-code** [OpenStitching/stitching#218] [Q&A]
I was wondering if there is a max number of stitched images I can run at once to better perform the stitching and if there is a way I can make this number bigger. Any help is great!

**Comment by lukasalexanderweber** [comment]
There is no hard maximum. What is your use case?

**Reply by davidhardeniv-code** [reply]
I did a drone flight, I am pretty much trying to save the drone flight. I took the video and converted it into images for every frame during the flight. Right now I am trying to stitch 300 images, but they are all very similiar. Currently I just took like every 30 images and stitched 10 togther but I was wondering if there was a wway to set if the images are pretty much the same could I just skip stitching that image and instead move onto the next. Also just if I was able to run through a lot of images in one run.

**Reply by lukasalexanderweber** [reply]
Ok but helping here is out of scope of the discussion section, sorry
